src/components/pages/Offers/Offers.jsx

```diff
@@ -33,11 +33,15 @@ class Offers extends PureComponent {
       name: nameKeywords,
       offererId,
       page,
+      soldOut,
+      expired,
       venueId: selectedVenueId,
     } = translateQueryParamsToApiParams(props.query.parse())
 
     const isFilteredByActiveStatus = active === EXCLUDING_STATUS_VALUE
     const isFilteredByInactiveStatus = inactive === EXCLUDING_STATUS_VALUE
+    const isFilteredBySoldOutStatus = soldOut === EXCLUDING_STATUS_VALUE
+    const isFilteredByExpiredStatus = expired === EXCLUDING_STATUS_VALUE
```

Comment:
`isFilteredBy{status}` devrait etre nommé  `is{status}Exclude`, ou alors changer `===` par `!==`

```suggestion
   // in _constants
    export const STATUS_FILTER_ACTIVE = 'active'
    export const STATUS_FILTER_INACTIVE = 'inactive'
    export const STATUS_FILTER_SOLD_OUT = 'sold_out'
    export const STATUS_FILTER_EXPIRED = 'expired'

    // in Offers
    import {
      STATUS_FILTER_ACTIVE,
      STATUS_FILTER_INACTIVE,
      STATUS_FILTER_SOLD_OUT,
      STATUS_FILTER_EXPIRED,
    } from './_constants'
    const statusFilters = {
      [STATUS_FILTER_ACTIVE]: active !== EXCLUDING_STATUS_VALUE,
      [STATUS_FILTER_INACTIVE]: inactive !== EXCLUDING_STATUS_VALUE,
      [STATUS_FILTER_SOLD_OUT]: soldOut !== EXCLUDING_STATUS_VALUE,
      [STATUS_FILTER_EXPIRED]: expired !== EXCLUDING_STATUS_VALUE,
    }

    this.state = {
      // [...]
      statusFilters,
      isFilteredByStatus: Object.values(statusFilters).includes(true)
      // [...]
    }
```